js-api-doc/legacy/options.d.ts

```diff
@@ -508,6 +508,27 @@ export interface LegacySharedOptions<sync extends 'sync' | 'async'> {
    * @compatibility dart: "1.43.0", node: false
    */
   logger?: Logger;
+
+  /**
+   * If this option is set to `'node'`, Sass will use the built-in Node.js
+   * package importer to resolve Sass files with a `pkg:` URL scheme. Details
+   * for library authors and users can be found in the
+   * {@link NodePackageImporter} documentation.
+   *
+   * @example
+   * ```js
+   * sass.renderSync({
+   *   data: '@use "pkg:vuetify";',
+   *   pkgImporter: {type: 'node'}
+   * });
+   * ```
+   * @category Plugins
+   * @compatibility dart: "2.0", node: false
+   */
+  pkgImporter?: {
+    type: 'node';
+    entryPointPath?: string;
+  };
```

Comment:
As per my comment on the other PR, I think this should just take an instance of `NodePackageImporter`.

Reply:
Updated in ac235148468a3778500a37d757bef8236616b0de and dbf2e85ea705a040b9dc506b84d71cb709180b20